src/transformers/integrations/integration_utils.py

```diff
@@ -933,6 +984,36 @@ def on_predict(self, args, state, control, metrics, **kwargs):
             metrics = rewrite_logs(metrics)
             self._wandb.log(metrics)
 
+    def on_evaluate(self, args, state, control, **kwargs):
+        if os.getenv("WANDB_LOG_EVALS"):
+            eval_loop_output = self.trainer.eval_loop_output
```

Comment:
Where is `eval_loop_output` coming from exactly?

Reply:
Thanks for catching, I missed this commit.  It's added here:
https://github.com/huggingface/transformers/pull/31050/commits/b8d5c6ee12939d80d99f40c022b5c781ea10c4d9